Add composeSchema helper (#3822)
plone/volto: docs/source/blocks/extensions.md

it for the `variation` extension.
```

> Volto provides a helper to combine multiple `schemaEnhancer` into a single

Reviewer: "objects" might not be the correct word, perhaps "extensions" or "extension mechanisms", following the pattern from the previous note? Or we could get technical and call it "thingies". 😜 

```suggestion
Volto provides a helper to combine multiple `schemaEnhancer` objects into a single
```

> function. This allows creating clean, single purpose reusable schema enhancers:

Reviewer: ```suggestion
function. This allows creating clean, single purpose, reusable schema enhancers:
```

it for the `variation` extension.
```

> Volto provides a helper to combine multiple schemaEnhancers into a single

Reviewer: It is confusing to have a mix of "schemaEnhancers" and "schema enhancers". In narrative text, I think the latter is preferred, whereas in code the former should be an inline literal.

```suggestion
Volto provides a helper to combine multiple schema enhancers into a single
```

function, so that multiple schema enhancers can be used. For example:

> ```

Reviewer: I'm not sure whether the syntax is js or jsx.

````suggestion
```js
````
